server/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "ToDo app API",
   "main": "build",
   "config": {
-    "test_port": "8000"
+    "test_port": "6000"
```

Comment:
1. Run `yarn test` - everything is ok
2. Change code slightly (add space somewhere and remove it)
3. The port is free
4. Run `yarn test` - error, because port stopped being empty right after starting the command:

![image](https://user-images.githubusercontent.com/3635774/50514621-61a16600-0ab0-11e9-9488-8fb8d64bb1a2.png)

So, the test server process was killed after running dredd, but it started to compile code before explicit start.

5. List of other ports works well, for ex 6000

Reply:
@AleksSenkou I've tested. This ↓ should resolve it:
```
"test:api": "yarn dev:test:kill; yarn test:dredd; yarn dev:test:kill",
```

Reply:
or you should be attentive with your test server 😉

Reply:
@AleksSenkou and there's an interesting fact for `server-wait: 3`( inside _dredd.yml_). It doesn't delay server up, it delays start of running test suite after server is up already.

Reply:
> @AleksSenkou I've tested. This ↓ should resolve it:
> 
> ```
> "test:api": "yarn dev:test:kill; yarn test:dredd; yarn dev:test:kill",
> ```

@DDKatch that didn't work for me

Reply:
> @AleksSenkou and there's an interesting fact for `server-wait: 3`( inside _dredd.yml_). It doesn't delay server up, it delays start of running test suite after server is up already.

Does that means that we can remove delay at all?

Reply:
@AleksSenkou to be honest there's half of true. What happens really is:
start running server -> wait for 3 secs (in our case we can wait 0 and it still works because we're using ssd and it's really tiny for now)-> server is fully up (in average) -> run test suite (aka queries)

Reply:
@DDKatch yeah, thank you for notes 👍 , I haven't spent time for better understanding of steps, but we have 3-4 seconds faster dredd circle right now, that's awesome